release-notes/3.0/preview/api-diff/preview7/standalone-packages/3.0-preview7-standalone-packages_System.IO.Pipelines.md

```diff
@@ -0,0 +1,30 @@
+# System.IO.Pipelines
```

Comment:
LGTM, but @davidfowl @jkotalik @Tratcher @halter73 should take a glace.

Reply:
Yeah looks good.